shared/actions/fs/index.js

```diff
@@ -75,6 +75,19 @@ const filePreview = (state: TypedState, action) =>
     )
     .catch(makeRetriableErrorHandler(action))
 
+const loadUserFileEdits = (state: TypedState, action) =>
+  RPCTypes.SimpleFSSimpleFSUserEditHistoryRpcPromise()
+    .then(writerEdits => {
+      const tlfUpdates = Constants.userTlfHistoryRPCToState(writerEdits || [])
+      Saga.all(tlfUpdates.map(u =>
```

Comment:
i think this line does nothing

Reply:
Yeah it's fixed in the next PR.

Reply:
Oh I see what you mean. I guess I need to chain it.